examples/adc_values.rs

```diff
@@ -79,7 +79,7 @@ fn SysTick() {
         if let Some(ref mut shared) = SHARED.borrow(cs).borrow_mut().deref_mut() {
             // Read temperature data from internal sensor using ADC
             let t = hal::adc::VTemp::read(&mut shared.adc, None);
-            writeln!(shared.tx, "Temperature {}.{}C\r", t / 100, t % 100).ok();
+            writeln!(shared.tx, "Temperature {}.{}C\r", t / 10, t % 10).ok();
```

Comment:
What's this? Is that a spurious change or a fix?

Reply:
Oh yeah whoops that's supposed to be in my other PR #168.